src/renderers/WebGLRenderer.js

```diff
@@ -1189,13 +1188,11 @@ function WebGLRenderer( parameters = {} ) {
 
 		if ( _transmissionRenderTarget === null ) {
 
-			const renderTargetType = ( isWebGL2 && _antialias === true ) ? WebGLMultisampleRenderTarget : WebGLRenderTarget;
-
-			_transmissionRenderTarget = new renderTargetType( 1, 1, {
+			_transmissionRenderTarget = new WebGLRenderTarget( 1, 1, {
 				generateMipmaps: true,
-				type: HalfFloatType,
+				type: utils.convert( HalfFloatType ) !== null ? HalfFloatType : UnsignedByteType,
```

Comment:
Is this revert intentional?

Reply:
Yes. Since we want to support transmission with WebGL 1, I throught it would be better to restore the original type setup of the render target.

Reply:
Sounds good 👍